examples/mobile/HomeApp/js/app.js

```diff
@@ -171,7 +218,16 @@ import Storage from "./clipping-storage.js";
 		burgerButton.addEventListener("click", onButtonClick);
 		popupButton.addEventListener("click", onPopupSubmit);
 
-		loadWebClipList();
+		appsList = storage.readAllFromStorage(Storage.elements.APPSLIST);
+
+		//TODO: all webclips url should checked if they are valid
+
+		if (!Array.isArray(appsList) || !appsList.length) {
```

Comment:
And remove condition `!Array.isArray(appsList)`

Reply:
done